popolo/importers/popit.py

```diff
@@ -109,12 +109,9 @@ def update_optional_area(object_data):
 
         # Create all areas:
         if 'areas' in data:
-            area_id_to_django_object = {}
             for area_data in data['areas']:
                 with show_data_on_error('area_data', area_data):
-                    popit_id, area = \
-                        self.update_area(area_data)
-                    area_id_to_django_object[popit_id] = area
+                    self.update_area(area_data)
```

Comment:
Sorry, I might have been being unclear in my comment, but I think you still want to have:
```
area_id, area = self.update_area
area_id_to_django_object[area_id] = area
```
... here, since when the parent-setting bit later is fixed, it'll need `area_id_to_django_object` to have every known area in it.